### PR TITLE
Adding GSJUG to Jakarta NoSQL specification

### PR DESCRIPTION
**Pull request by mpredli01**
Signed-off-by: Michael P. Redlich <mike@redlich.net>

The Garden State Java User Group would like to adopt the Jakarta NoSQL specification.

### CODE REVIEW

**Comment by mpredli01** [comment]
How do I proceed with fixing these errors?

File: data/spec_adopters.yml [hunk 1]
```diff
@@ -97,7 +97,16 @@ adopters:
       homepage_url: https://twitter.com/TurkeyJBUG/
       logo: turkeyjboss.png
       logo_white: turkeyjboss.png
-      
+
+ - name: Jakarta NoSQL
```
Comment (ivargrimstad) [inline data/spec_adopters.yml:101]: The joys of YAML. You need to fix the indentation here, so it is aligned with the others.

File: jakarta.ee.iml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ivargrimstad) [inline jakarta.ee.iml:1]: Delete this file

File: .idea/vcs.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ivargrimstad) [inline .idea/vcs.xml:1]: Delete this file

File: .idea/modules.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ivargrimstad) [inline .idea/modules.xml:1]: Delete this file

File: .idea/misc.xml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ivargrimstad) [inline .idea/misc.xml:1]: Delete this file

File: .idea/compiler.xml [hunk 6]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ivargrimstad) [inline .idea/compiler.xml:1]: Delete this file

File: .idea/.gitignore [hunk 7]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```
Comment (ivargrimstad) [inline .idea/.gitignore:1]: Delete this file

**Review by ivargrimstad — CHANGES_REQUESTED** [review]
See my comments. The only files in the PR should be the data/spec_adopters.yml and the logo file.

**Review by autumnfound — CHANGES_REQUESTED** [review]
The logo image should be compressed to be under 50kb, if you need help with this I can make a quick commit for that.

File: data/spec_adopters.yml (hunk 1 above)
Comment (mpredli01) [reply data/spec_adopters.yml:101]: Hi Ivar:

Thanks for reviewing and finding these errors.  I could have sworn I had everything lined-up nicely in IntelliJ IDEA.  Anyway, I re-cloned the repository and started fresh.  I also didn't use IDEA this time - sometimes vi is the best choice!  :-)

I will be submitting a new pull request...

Mike.

**Comment by mpredli01** [comment]
> The logo image should be compressed to be under 50kb, if you need help with this I can make a quick commit for that.

Hi @autumnfound:

Thanks for letting me know about the size restriction.  I will be submitting a new pull request with a logo less than 50KB.

Mike.

**Pull request closed without merging**